### PR TITLE
Added ORM

### PR DESCRIPTION
**Pull request by GreenDragonBG**

### CODE REVIEW

File: .env
Comment (AlexOgnyanov) [inline .env:1]: Remove .env from

File: src/setup-model.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        }
+    },
+);
+
+export default User;
+
+
+
+
+
```
Comment (AlexOgnyanov) [inline src/setup-model.js]: Remove empty lines

**Review by AlexOgnyanov — CHANGES_REQUESTED** [review]

File: .env
Comment (GreenDragonBG) [reply .env:1]: Done

File: src/setup-model.js (hunk 1 above)
Comment (GreenDragonBG) [reply src/setup-model.js]: Done

**Pull request merged**